language/stdlib/modules/event.mvir

```diff
@@ -0,0 +1,92 @@
+// This module is used for emitting events into the event_store.
```

Comment:
We probably want a much much more in depth comment than this. Or at least something that points to a doc.
These core modules are going to be one of the first stepping stones for anyone trying to learn Move and it's environment on chain. 
Imagine never having encountered Events before. The comments and documentation in this file need to be sufficient enough to get that person started. 
And unlike LibraCoin and LibraAccount, I feel like much of this is much more difficult to explain as it isn't quite as "core" to Libra